config/env/all.js

```diff
@@ -5,5 +5,5 @@ var keys = rootPath + '/keys.txt';
 module.exports = {
 	root: rootPath,
 	port: process.env.PORT || 3000,
-  db: process.env.DB_URL
+    db: process.env.MONGOHQ_URL
```

Comment:
Yes, in my .env file for the branch, the mongodb database url is set on the variable MONGOHQ_URL and not on DB_URL and as such when i run npm start to start the project, the right database url to connect to isn't found, so I changed it in the branch for the coverage setup and I forgot to change it in this branch for front end unit setup. Do I need to cancel this particular PR, and also effect it here?